spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/SpatialDropout3D.scala

```diff
@@ -29,19 +29,16 @@ class SpatialDropout3D[T: ClassTag](val p: Double = 0.5,
                                    )(implicit ev: TensorNumeric[T])
   extends KerasLayer[Tensor[T], Tensor[T], T](KerasLayer.addBatch(inputShape)) {
 
+  require(format == "CHANNEL_FIRST" || format == "CHANNEL_LAST", s"$format is not supported")
```

Comment:
Add `toLowerCase()`, just allow users to input lower case `channel_last`

Reply:
refer to https://github.com/intel-analytics/BigDL/pull/2220/files#diff-b2505a56444fff9b4bdd5ae73b9f943dR32